data/typelibrary/net-1.0.0/typelib/CLIENT_2_1.fbt

```diff
@@ -1,41 +1,42 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<!DOCTYPE FBType SYSTEM "http://www.holobloc.com/xml/LibraryElement.dtd">
-<FBType Comment="" Name="CLIENT_2_1">
-  <Identification Description="Copyright (c) 2017 fortiss GmbH&#13;&#10; &#13;&#10;This program and the accompanying materials are made&#13;&#10;available under the terms of the Eclipse Public License 2.0&#13;&#10;which is available at https://www.eclipse.org/legal/epl-2.0/&#13;&#10;&#13;&#10;SPDX-License-Identifier: EPL-2.0" Standard="61499-2"/>
-  <VersionInfo Author="Alois Zoitl" Date="2017-10-25" Organization="fortiss GmbH" Remarks="initial API and implementation and/or initial documentation" Version="1.0"/>
-  <InterfaceList>
-    <EventInputs>
-      <Event Comment="" Name="INIT" Type="Event">
-        <With Var="QI"/>
-        <With Var="ID"/>
-      </Event>
-      <Event Comment="" Name="REQ" Type="Event">
-        <With Var="QI"/>
-        <With Var="SD_1"/>
-        <With Var="SD_2"/>
-      </Event>
-    </EventInputs>
-    <EventOutputs>
-      <Event Comment="" Name="INITO" Type="Event">
-        <With Var="QO"/>
-        <With Var="STATUS"/>
-      </Event>
-      <Event Comment="" Name="CNF" Type="Event">
-        <With Var="QO"/>
-        <With Var="STATUS"/>
-        <With Var="RD_1"/>
-      </Event>
-    </EventOutputs>
-    <InputVars>
-      <VarDeclaration Comment="" Name="QI" Type="BOOL"/>
-      <VarDeclaration Comment="" Name="ID" Type="WSTRING"/>
-      <VarDeclaration Comment="" Name="SD_1" Type="ANY"/>
-      <VarDeclaration Comment="" Name="SD_2" Type="ANY"/>
-    </InputVars>
-    <OutputVars>
-      <VarDeclaration Comment="" Name="QO" Type="BOOL"/>
-      <VarDeclaration Comment="" Name="STATUS" Type="WSTRING"/>
-      <VarDeclaration Comment="" Name="RD_1" Type="ANY"/>
-    </OutputVars>
-  </InterfaceList>
+<?xml version="1.0" encoding="UTF-8"?>
+<FBType Name="CLIENT_2_1" Comment="Connect to a SERVER_1_2 Block">
+	<Identification Standard="61499-2" Description="Copyright (c) 2017 fortiss GmbH&#10; &#10;This program and the accompanying materials are made&#10;available under the terms of the Eclipse Public License 2.0&#10;which is available at https://www.eclipse.org/legal/epl-2.0/&#10;&#10;SPDX-License-Identifier: EPL-2.0" >
+	</Identification>
+	<VersionInfo Organization="fortiss GmbH" Version="1.0" Author="Alois Zoitl" Date="2017-10-25" Remarks="initial API and implementation and/or initial documentation">
+	</VersionInfo>
+	<InterfaceList>
+		<EventInputs>
+			<Event Name="INIT" Type="Event" Comment="Open new connection">
```

Comment:
With init you can also close the connection, INITO reports on (de-)initialization success

Reply:
yes, close then needs to set QI = FALSE. right ? 

INIT, QI = TRUE: Open Connection, 

INIT, QI = FALSE: Close Connection

Reply:
should we make a long comment here ? 

Open new connection (QI = TRUE) / Close connection (QI = FALSE)

or go with like: 

Open new connection / Close connection

Reply:
I like the 

 Open new connection (QI = TRUE) / Close connection (QI = FALSE)

Version.